### PR TITLE
[MINOR][DOCS] Fix some spelling typos

### PR DESCRIPTION
**Pull request by YuanHanzhong**
### What changes were proposed in this pull request?
Fixed typo.

### Why are the changes needed?
To help make spark perfect.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
Just comment typo, no need to test.

### Was this patch authored or co-authored using generative AI tooling?
No.

### CODE REVIEW

**Comment by YuanHanzhong** [comment]
Fixe two typoes, let's make spark perfect together.

**Comment by yaooqinn** [comment]
Hi @YuanHanzhong, thanks for your first contribution to spark. 

However, could you please include #43636 in this patch as well? Also, kindly answer the questions in the PR template instead of removing them all.

File: sql/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This module provides support for executing relational queries expressed in eithe
 Spark SQL is broken up into four subprojects:
  - Catalyst (sql/catalyst) - An implementation-agnostic framework for manipulating trees of relational operators and expressions.
  - Execution (sql/core) - A query planner / execution engine for translating Catalyst's logical query plans into Spark RDDs.  This component also includes a new public interface, SQLContext, that allows users to execute SQL or LINQ statements against existing RDDs and Parquet files.
- - Hive Support (sql/hive) - Includes extensions that allow users to write queries using a subset of HiveQL and access data from a Hive Metastore using Hive SerDes. There are also wrappers that allow users to run queries that include Hive UDFs, UDAFs, and UDTFs.
+ - Hive Support (sql/hive) - Includes extensions that allow users to write queries using a subset of HiveSQL and access data from a Hive Metastore using Hive SerDes. There are also wrappers that allow users to run queries that include Hive UDFs, UDAFs, and UDTFs.
```
Comment (yaooqinn) [inline sql/README.md:9]: From what I have gathered, HiveQL appears to be a widely accepted term. e.g. https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/EMRforDynamoDB.ProcessingHiveQL.html

Comment (maintian) [reply sql/README.md:9]: Yes I agree, HiveQL should be the accepted term

Comment (YuanHanzhong) [reply sql/README.md:9]: Yes, I learned, thank you for correcting me.

**Comment by YuanHanzhong** [comment]
> Hi @YuanHanzhong, thanks for your first contribution to spark.
> 
> However, could you please include #43636 in this patch as well? Also, kindly answer the questions in the PR template instead of removing them all.

Thanks for tell me, I corrected me mistakes. I will answer the questions in the next PR .

**Comment by yaooqinn** [comment]
> I will answer the questions in the next PR.

Hi, please fix the PR description for this one.

**Comment by YuanHanzhong** [comment]
Any other suggestions?

**Review by yaooqinn — APPROVED** [review]

**Comment by yaooqinn** [comment]
Thanks @YuanHanzhong, merged to master

**Comment by YuanHanzhong** [comment]
I appreciate your help, have a nice day!

**Pull request closed without merging**